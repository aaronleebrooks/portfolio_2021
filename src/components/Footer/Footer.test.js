import { render, screen } from '@testing-library/react';
import Footer from './Footer';

describe('Footer', () => {
  test('renders', () => {
    const {container} = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
})
