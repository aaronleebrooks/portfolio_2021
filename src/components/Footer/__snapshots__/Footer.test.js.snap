// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders 1`] = `
<div>
  <div>
    This is a footer
  </div>
</div>
`;
